www/src/utils/tokens/colors.js

```diff
@@ -1,42 +1,150 @@
-import gray from "gray-percentage"
+const palette = {
+  purple: {
+    900: `#362066`,
+    800: `#452475`,
+    700: `#542C85`,
+    600: `#663399`,
+    500: `#8a4baf`,
+    400: `#B17ACC`,
+    300: `#D9BAE8`,
+    200: `#F1DEFA`,
+    100: `#F6EDFA`,
+    50: `#FCFAFF`,
```

Comment:
Overall this is looking good! I know that some people use the 100s to account for "in-between" colors later on, but I find it a little bit odd since I'd never want e.g. 100 shades of purple and 10 shades seems like enough variation – not at all blocking, just my two cents

Reply:
Great 2 cents! 🙏

I thought a bit more about this while grocery shopping, and I'll go ahead and refactor to drop one digit — `5`-`90` seems like it is a good fit irt the brightness of the upper and lower end of the current palette (even if I like `10`-`100` more); "German Angst" keeps me from going super minimal (1-10) 😅, … and at least for now some of the darker "secondary palette" colors like yellow or orange don't have a fitting proper text color share, which would make a nice "100" … "0" could be white on the `gray` scale, and `100` black, … not that I _really_ want to … yada yada yada 🤷‍♂ 😉 

FWIW, I think I should share that the palette IMO is way to big for "just" UI anyway, lots of not-really-needed secondary colors in there. I started out with just a handful of accent colors and 2-4 shades for each, but eventually expanded those color ranges to the full ten because I wanted a palette that could also be used for illustrations/carrier elements. Colors are hard … and this is the easier part still, right? 😸